netdiff/parsers/openvpn.py

```diff
@@ -0,0 +1,67 @@
+import networkx
+from openvpn_status import ParsingError, parse_status
+
+from ..exceptions import ConversionException, ParserError
+from .base import BaseParser
+
+
+class OpenvpnParser(BaseParser):
+    """ OpenVPN status log parser """
+    protocol = 'openvpn'
+    version = ''
```

Comment:
the status log file of openvpn has 2 versions, we could use this attribute to indicate which version we support